src/styles/blocks/cards-cointainer.scss

```diff
@@ -0,0 +1,19 @@
+.cards-cointainer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: center;
+
+  &__card {
+    margin-right: 48px;
+    margin-top: 49px;
```

Comment:
You can add padding directly to the container rather than setting margins for the cards. As well you can set a width of the container to limit it space and justify content in a way so it create equal gaps between cards